Readme.md

```diff
@@ -7,7 +7,7 @@ For LabVIEW NXG versions see the [previous releases](https://github.com/ni/webvi
 
 The following examples demonstrate different ways of using WebVIs. Each directory contains example code and a `README.md` file explaining the example.
```

Comment:
It looks like the following wind turbine url is working: `https://ceoe-contrail.ceoe.udel.edu:8100/Mjpeg/1?authToken=d392c0a2-3a6c-4a94-aad8-1c248e44915a`

Seen here: https://publicutility.ceoe.udel.edu/lewesturbine/multimedia.shtml#live

Can you update the example to use that url, push the webvi changes (but not Build changes, those should be a follow-up PR), and update the GIF?